File: app/templates/server_node/koa/server_node_koa/api/todo/routes/todo-routes.js
"use strict";

const TodoController = require('../controller/todo-controller');

module.exports = class TodoRoutes {
    static init(router) {
      let _todoController = new TodoController();

      router
        .get('/api/todos', _todoController.getAll)
        .post(TodoController.createTodo);

      router
        .delete('/api/todos/:id', _todoController.deleteTodo);
    }
}

File: app/templates/server_go/echo/api/todo/routes/todoroutes.go
package todoroutes

import (
	"<%= repoHostUrl %>/<%= userNameSpace %>/<%= appName %>/server/api/todo/controller"
	"github.com/labstack/echo"
)

func Init(e *echo.Echo) {
	e.GET("/api/todos", todocontroller.GetAll)
	e.POST("/api/todos", todocontroller.NewTodo)
	e.DELETE("/api/todos/:id", todocontroller.RemoveTodo)
}
